website/blog/flink-native-support/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Data lineage for Apache Flink
+date: 2024-11-15
+authors: [Leszczynski]
+description: Flink native lineage support will bring OpenLineage integration to the new era.
+---
+Apache Flink is a powerful stream processing engine widely adopted in the industry.
+Flink jobs are capable of emitting OpenLineage events for already two years.
+However, it is a recent joint effort of Flink and OpenLineage communities that will bring this integration into the new era.
+
+<!--truncate-->
+
+### Existing OpenLineage Flink integration
+
+As of November 2024, [OpenLineage Flink integration](https://github.com/OpenLineage/OpenLineage/tree/main/integration/flink) supports Flink versions `1.15` to `1.18` and emits OpenLineage events complying with the dataset naming model.
+It supports Kafka sources and sinks with Avro and Protobuf, JDBC and Cassandra connectors with extracting lineage from the queries executed.
+It also supports Hybrid sources. Overall, this is extremely helpful for the organizations to collect consistent cross-platform lineage graphs.
```

Comment:
`this is extremly` -> `it is extremly`

Reply:
also for now it seems to me to say that "In general its helpful for the organisation to have have cross platform lineage" and I think you wan to say that "these features can aid organisations greatly in lineage collection"